rules.md

```diff
@@ -0,0 +1,18 @@
+# Substrate
+
+- 🔒️ed lines are reviewed by 2 team leads, one of them must be in @paritytech/locks-review and the other in @paritytech/polkadot-review.
+- Normal PRs are reviewed by 2 of @paritytech/core-devs. This group should not be explicitly mentioned, but the check should be fail explaining this rule.
```

Comment:
Replace "be fail" with "fail" everywhere

Replace "This group should not be explicitly mentioned" with "This team's reviews should not be requested"

So I take we'll not be doing https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#configuring-auto-assignment.

Reply:
I might have misinterpreted "mentioned" (pr-custom-review doesn't mention anything) since I assumed it's "requested". Is this related to https://github.com/paritytech/substrate/pull/10951#discussion_r827314600? There I explained that not everyone should be pinged for each PR given that https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#configuring-auto-assignment is properly configured.

Reply:
> So I take we'll not be doing https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#configuring-auto-assignment for `core-devs`

no, at least not yet. `core-devs` is a very large group (and for me, it's still a question why wouldn't we create more narrow-specialized groups. But it's surely OK to start like this). And it doesn't really make sense to either ping them all or do the auto-assignment in that group.

Reply:
> and for me, it's still a question why wouldn't we create more narrow-specialized groups

I can see that being reasonable given that core-devs will not be a blocker for processbot anymore after pr-custom-review is applied.

> it doesn't really make sense to either ping them all or do the auto-assignment in that group

Alright